tests/test_common/test_setup.py

```diff
@@ -66,6 +66,10 @@ def test_from_config_raises(self):
         with self.assertRaises(MinosConfigNotProvidedException):
             _MinosSetupMock.from_config()
 
+    def test_del(self):
```

Comment:
Should we complete this test? If not maybe further documentation may be nice so we know in the future what to implement.

Reply:
Solved!